src/rpc/connection.rs

```diff
@@ -216,52 +214,48 @@ impl Connection {
 				biased;
 				// Check if this has shutdown
 				_ = canceller.cancelled() => break,
+				// Remove any completed tasks
+				Some(_) = tasks.join_next() => continue,
```

Comment:
we are ignoring the potential errors returned by the spawned task. Is it intended?

Reply:
OK, updated. We now exit if there is an uncaught panic in an async task.